src/controllers/categories.js

```diff
@@ -22,17 +22,15 @@ const categoriesByGroup = async (req, res) => {
   res.status(201).json(result);
 };
 
-
-// TODO feat controllers
 const createCategory = async (req, res) => {
     
-  const result = await Category.create(req.body);
+  const result = await Category.create(req.query);
```

Comment:
The previous solution with req.body was better, because in POST methods we should pass data in body

Reply:
But it was not working with "body" and with "query" works. At list in Postman I was testing

Reply:
It probably didn't work, because you tried to pass data in `Params` tab instead of passing JSON in `Body` tab

Reply:
Yes, thank you. The mistake was in using wrong tabs in Postman.